SetupBLT.cmake

```diff
@@ -193,8 +193,10 @@ if (NOT BLT_LOADED)
     # File extension lists used to filter sources based on languages for code checks
     # and filtering Fortran sources out of cuda/hip source lists
     # Note: this filtering is case-insensitive
-    set(BLT_C_FILE_EXTS ".cpp" ".hpp" ".cxx" ".hxx" ".c" ".h" ".cc" ".hh" ".inl" ".cu" ".cuh"
-               CACHE STRING "List of known file extensions used for C/CXX sources")
+    set(BLT_C_FILE_EXTS ".c" ".h"
```

Comment:
We need this in the release notes. Also the fact we require the hip compiler to not be the main compiler anymore.